quickget

```diff
@@ -1023,6 +1038,30 @@ function get_opensuse() {
     make_vm_config "${ISO}"
 }
 
+function get_oraclelinux() {
+    local HASH=""
+    local ISO=""
+    local URL=""
+
+    local arch="x86_64"
```

Comment:
I wonder if `arch="$(arch)"` would be useful

Reply:
> I wonder if `arch="$(arch)"` would be useful

Yes, perhaps, but my intention was to limit the scope of this first PR just to keep things simple. Is this something that could wait to be added later?

Reply:
could be - although it might be more trouble than help.  I'd  mentally file it for if/when the question arises in earnest.